research-software-directory/RSD-as-a-service
# Implement SonarCloud suggestions

# SonarCloud improvements

Changes proposed in this pull request:

* Implement some of the suggestions from [SonarCloud](https://sonarcloud.io/summary/overall?id=nl.research-software%3Aauthentication) for the authentication module
* Add to the readme that you should relax your firewall when developing the auth module locally

**Note:** Since this PR is already big enough, I will look at further improvements later.

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up`
* Login using the various login providers, this should work without errors

Related to #915

PR Checklist:
